daemon/compartment.c

```diff
@@ -152,6 +152,34 @@ enum compartment_start_sync_msg {
 
 static list_t *compartment_module_list = NULL;
 
+bool
+compartment_is_stoppable(compartment_t *compartment)
+{
+	if ((compartment_get_state(compartment) == COMPARTMENT_STATE_STARTING) || (compartment_get_state(compartment) == COMPARTMENT_STATE_RUNNING) || (compartment_get_state(compartment) == COMPARTMENT_STATE_BOOTING	) /*|| 
```

Comment:
just run './scripts/format-code.sh' , this is also helpful for better reading code during review.